packages/components/stories/read-more/README.md

```diff
@@ -0,0 +1,43 @@
+# ReadMore component
+
+Renders a `ReadMore` component which use to collapse and expand long text.
+
+#### Supported Gestures:
+
+-   Click
+
+## Usage
+
+```jsx
+import { ReadMore } from 'deriv-components';
+
+const DummyComponent = props => <ReadMore 
+                                    expand_text={'See more'} 
+                                    text={'long text which you want to show by read more' }
+                                    collapse_length= {20}
+                                    className= {'ClassName'}
+                                />;
+```
+
+## Props
+
+| Name            | Type      | Default | Description                                                         |
+| ----------------| --------- | ------- | ------------------------------------------------------------------- |
+| expand_text     | {string}  | null    |                                                                     |
+| text            | {string}  | null    |                                                                     |
+| collapse_length | {number}  | null    |                                                                     |
```

Comment:
Could we assign better default values?

Reply:
Actually, for now, it works without default values if we'll update the component we could assign better default values